### ISSUE TITLE
Error EAI_AGAIN lemmy after upgrade

### ISSUE THREAD

**Issue by skariko** [LemmyNet/lemmy-ansible#47]
Hello,
I've tried to update from 0.16.3 to 0.16.5 with ansible.

It is on my test instance, i use this:

`git pull`

and

`ansible-playbook -vvv -u MYUSERNAME -k -K -i inventory/hosts lemmy.yml --become`

It does everything with no error but when I try to visit the website i receive:

`404: FetchError: request to http://lemmy:8536/api/v3/site? failed, reason: getaddrinfo EAI_AGAIN lemmy`

It's just my problem? What could have gone wrong?

**Comment by skariko** [comment]
I see that after `git pull` the version on file VERSION says: `0.16.4` but we are at 0.16.5.

Just saying, don't know if it is correlate with the error.

**Comment by dessalines** [comment]
Did you search for that error? Seems to be a DNS issue, or maybe a problem with docker. 

0.16.5 isn't released yet, its a minor bugfix.

**Comment by skariko** [comment]
> Did you search for that error? Seems to be a DNS issue, or maybe a problem with docker.
> 
> 0.16.5 isn't released yet, its a minor bugfix.

Found out just now what the problem was.

I am writing this to help others:
the federation was active in that instance but it was also marked as a private instance. This gave no problems in using it until the update (probably because it is restarted and re-run Docker).

I removed the federation from the config file leaving only the private instance and it restarted immediately.

Thank you.
